source/server/listener_manager_impl.h

```diff
@@ -152,6 +152,12 @@ class DrainingFilterChainsManager {
     drain_timer_->enableTimer(drain_time);
   }
 
+  void insertFilterChain(const Network::FilterChain* filter_chain) {
```

Comment:
s/insertFilterChain/addFilterChainToDrain (or similar) and please add doc comments.